auth/tenant_management.js

```diff
@@ -1,17 +1,81 @@
 const admin = require('firebase-admin');
 admin.initializeApp();
 
+const tenantId = 'tenantId';
+
+function createTenant() {
+  // [START auth_create_tenant]
+  admin.auth().tenantManager().createTenant({
+    displayName: 'myTenant1',
+    emailSignInConfig: {
+      enabled: true,
+      passwordRequired: false, // Email link sign-in enabled.
+    },
+  })
+  .then((createdTenant) => {
+    console.log(createdTenant.toJSON());
+  })
+  .catch((error) => {
+    // Handle error.
+  });
+  // [END auth_create_tenant]
+}
+
+function updateTenant() {
+  // [START auth_update_tenant]
+  admin.auth().tenantManager().updateTenant(tenantId, {
+    displayName: 'updatedName',
+    emailSignInConfig: {
+      enabled: false, // Disable email provider.
+    },
```

Comment:
Originally, I had thought to just piggy-back the enablement of anon authentication docs into the existing create/update tenant docs. My reasoning was that this is just another parameter, not really much different than (eg) emailSignInConfig. If you did that here, you could get rid of the enableAnonAuth snippet entirely. (No objection to keeping it separate either though.)